### PR TITLE
Example fix

### PR DESCRIPTION
**Pull request by leafyoung**
- Added the alternative event filtering code to retrieve historical events.
- Updated the test script.

### CODE REVIEW

File: models/examples/e_05_contract.py [hunk 1]
```diff
@@ -43,16 +52,51 @@ def run(self, _) -> ExampleModelOutput:
...
+                toBlock=self.context.block_number
+            ).get_all_entries()
+
+            output.log_io(input="""
+    vesting_added_events = contract.events.VestingScheduleAdded.createFilter(
+                fromBlock=0,
+                toBlock=self.context.block_number
+            ).get_all_entries()""", output=vesting_added_events)
+        except (ValueError, socket.timeout, ReadTimeoutError, ReadTimeout):
+            # Some Eth node does not support the newer eth_newFilter method
```
Comment (newtectonics) [inline models/examples/e_05_contract.py:67]: yeouch

File: models/examples/e_05_contract.py [hunk 2]
```diff
@@ -43,16 +52,51 @@ def run(self, _) -> ExampleModelOutput:
...
-            toBlock=self.context.block_number
-        ).get_all_entries()""", output=vesting_added_events)
+        try:
+            vesting_added_events = contract.events.VestingScheduleAdded.createFilter(
+                fromBlock=0,
+                toBlock=self.context.block_number
+            ).get_all_entries()
+
+            output.log_io(input="""
+    vesting_added_events = contract.events.VestingScheduleAdded.createFilter(
```
Comment (newtectonics) [inline models/examples/e_05_contract.py:62]: soon we'll be able to do this via the views, rather than calling web3 events

**Review by newtectonics — DISMISSED** [review]

**Comment by newtectonics** [comment]
oopsies, some merge conflicts on the run.sh

File: models/examples/e_05_contract.py (hunk 2 above)
Comment (leafyoung) [reply models/examples/e_05_contract.py:62]: yeah. I read what Peter wrote about the views.

**Comment by leafyoung** [comment]
> GitHub's cryptic message: "[leafyoung] dismissed [newtectonics]’s stale review via 43ab892 8 minutes ago". How could I not dismiss?
I will break up run.sh.

**Pull request merged**